.github/dependabot.yml

```diff
@@ -307,6 +307,26 @@ updates:
     schedule:
       interval: "weekly"
       day: "sunday"
+  -
+    package-ecosystem: "gomod"
```

Comment:
just for me: what does this actually do?

Reply:
I think the Otel repo is using https://dependabot.com/